src/js/components/Widgets/BallotItemSupportOpposeCountDisplay.jsx

```diff
@@ -267,6 +267,14 @@ class BallotItemSupportOpposeCountDisplay extends Component {
   }
 
   render () {
+    const handleEnterHoverLocalArea = () => {
+      this.props.handleLeaveCandidateCard();
+    };
+
+    const handleLeaveHoverLocalArea = () => {
+      this.props.handleEnterCandidateCard();
```

Comment:
Since this function is coming in as a prop, and may or may not exist, you want to wrap it with:
```
if (this.props.handleEnterCandidateCard) {
  this.props.handleEnterCandidateCard();
}
```